raspi/access_point/server/server.py

```diff
@@ -47,6 +47,8 @@ def login():
     password = request.args.get('password')
     subprocess.call(['sudo', '-u', 'pi', susiconfig, 'set', "susi.mode="+auth, "susi.user="+email, "susi.pass="+password]) #nosec #pylint-disable type: ignore
     display_message = {"authentication":"successful", "auth": auth, "email": email, "password": password}
+    if auth == 'authenticated' and email != "":
+        os.system('echo "@reboot python3 /home/pi/SUSI.AI/susi_installer/raspi/access_point/register.py" | crontab -')
```

Comment:
Calling register at reboot via crontab we might have no network connection (which takes time via dhcp).

So it would be better to use a systemd unit that depends on network being up, which would guarantee that we can actually log in.

Furthermore, should this be run on *each* and *every* reboot later on? If yes, we need to have a logout functionality.

Reply:
@norbusan it works with crontab though maybe because registration tries to call the API three times before resetting(Intervals of 5 Seconds each). But you are right I will make a systemd service for it.

> Furthermore, should this be run on each and every reboot later on? If yes, we need to have a logout functionality.

No, it runs only after the first restart
Currently, this is done by removing crontab at the end of the `register.py` script.
Similarly, after I implement a systemd service, the script will just disable the service on completion.

Reply:
Indeed, I missed the crontab -r at the end.
And thanks for switching to systemd to ensure proper operation